src/control/lib/control/pool.go

```diff
@@ -1229,7 +1229,7 @@ func ListPools(ctx context.Context, rpcClient UnaryInvoker, req *ListPoolsReq) (
 			return nil, errors.New("pool query response uuid does not match request")
 		}
 
-		p.State = resp.State
+		p.State = resp.State.String()
```

Comment:
why does the state need to be held as a string here? is that inconsistent with PoolQueryResponse?

Reply:
Well this belongs to ListPoolsResp which is an array of pools and state is string, and I kept it as it is.
https://github.com/daos-stack/daos/blob/bfc2ee0bd7eddac3109bb556ec2ed799e2f480da/src/proto/mgmt/pool.proto#L147C34-L147C34